frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/basicComment/basicComment.component.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { TicketCommentHistoryBlock, TicketCommentReplyMetadata } from '@/v5/store/tickets/comments/ticketComments.types';
+import { editedCommentMessage } from '@/v5/store/tickets/comments/ticketComments.helpers';
+import { DialogsActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { getImgSrc } from '@/v5/store/tickets/tickets.helpers';
+import { CommentImages } from '../commentImages/commentImages.component';
+import { CommentMarkDown } from '../commentMarkDown/commentMarkDown';
+import { CommentContainer, CommentAge, CommentAuthor, EditedCommentLabel, SingleImage, CommentImagesContainer } from './basicComment.styles';
+import { CommentReply } from '../commentReply/commentReply.component';
+
+export type BasicCommentProps = {
+	images?: string[];
+	children?: any;
+	className?: string;
+	message?: string;
+	commentAge: string;
+	history?: TicketCommentHistoryBlock[];
+	metadata?: TicketCommentReplyMetadata;
+	isCurrentUserComment?: boolean;
+	author?: string;
```

Comment:
Should author be required?

Reply:
Nope, if you look at line 58, there is a conditional rendering, meaning the author may be empty.
This could happen if the user is from currentUser, if it deleted, or if it from otherUser, but it is not the first contiguous message from that author